data/ill_requests.json

```diff
@@ -0,0 +1,81 @@
+[
+  {
+    "status": "pending",
+    "copy": false,
+    "document": {
+      "title": "Histoire d'\u00c9gypte",
+      "authors": "Aly Badr, Cairo University",
+      "publisher": "Cairo University Press",
+      "year": "1920",
```

Comment:
I am not that old :-)

Reply:
It's not me ; these data come from UserStory fixtures ;-)